.travis.yml

```diff
@@ -22,4 +26,4 @@ script:
     - cargo build # Linter is also executed here
     - RUST_BACKTRACE=1 cargo test
     - jshint static/*.js test/selenium/*.js
-    - (cargo run &> /dev/null &) ; npm test
+    - (sudo ./target/debug/foxbox -n foxbox &> /dev/null &) ; npm test
```

Comment:
I'm not sure where cargo and rust is installed by Travis, but I get ```cargo: command is not found``` if I run ```cargo run``` with sudo, do you have an idea how to fix this in a nicer way?